content/ko/contact.md

```diff
@@ -22,33 +22,25 @@ menu:
 
 ## 메일링 리스트
 
-뉴스레터를 구독하려면[여기를 클릭하세요.](https://outreach.abetterinternet.org/l/1011011/2023-02-16/6l51)
+뉴스레터를 구독하려면[여기를 클릭하세요.](https://mailchi.mp/letsencrypt.org/fjp6ha1gad)
 
 ## 보안
 
 **Let's Encrypt와 관련된 보안 이슈 내용이 아니라면 이 주소로 메세지를 보내지 말아주세요.**
 
-<span id="email">이메일: </span>
-
-<script>
-  var parts = ["security", '@', "letsencrypt", ".", "org"];
-  var anchor = document.createElement("a");
-  anchor.href = "mailto:" + parts.join("");
-  anchor.text = parts.join("");
-  document.getElementById("email").appendChild(anchor)
-</script>
+이메일: [security@letsencrypt.org](mailto:security@letsencrypt.org)
 
 ### 안전한 통신
 
 보안 팀과 안전하게 통신하려면 GPG 키를 사용해 주십시오. 오프라인 마스터 키로 서명한 여러 하위 키로 구성됩니다. 모든 최신 버전의 GnuPG는 이러한 종류의 하이브리드 키를 지원합니다. 현재 키 구조는 다음과 같습니다.:
 
 ```
-pub   rsa4096 2015-11-24 [CE] [expires: 2025-09-25]
+pub   rsa4096 2015-11-24 [CE] [expires: 2023-09-18]
       0148 3B31 D8F9 DBA2 5D41  4DAA 718E 9F6D 10EC 230B
 uid           [ultimate] ISRG Security Team (letsencrypt.org) <security@letsencrypt.org>
-sub   rsa4096 2015-11-24 [E] [expires: 2023-09-25]
-sub   rsa4096 2015-11-24 [A] [expires: 2023-09-25]
-sub   rsa4096 2015-11-24 [S] [expires: 2023-09-25]
+sub   rsa4096 2015-11-24 [E] [expires: 2022-09-25]
+sub   rsa4096 2015-11-24 [A] [expires: 2022-09-25]
+sub   rsa4096 2015-11-24 [S] [expires: 2022-09-25]
```

Comment:
Looks like a regression

Reply:
> Looks like a regression

Nice spot @tdelmas . Than we have such problem across all languages. Because this comes from the source imported to Crowdin. An automated refresh of sources in Crowdin would solve that. Let me fix this file and we need to start thinking about a complete update of sources in Crowdin ASAP.

Reply:
This source file in Crowdin is 2 years old!! 
<img width="773" alt="Screenshot 2023-09-22 at 14 37 49" src="https://github.com/letsencrypt/website/assets/55971668/9e0e270b-ad7b-4722-91a6-4eaf71d2a8d6">

Reply:
@tdelmas  I will ask community in Crowdin to update translations for this file. There are other strings which need update here too. I could do a manual replacement but this is not a systematic change. Once the updates are ready I will push another update.

Reply:
Did it in Italian 👋🏻

Reply:
Ciao @albertopisaroni good job, you're first to fix it. https://github.com/letsencrypt/website/pull/1573